spec/currency_spec.rb

```diff
@@ -321,6 +321,16 @@ def to_s
       end
     end
 
+    describe "iso code" do
```

Comment:
please use the method name here (look at other describe blocks for examples), it improves navigation